torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -169,10 +169,17 @@ struct GraphFuser {
   Block* block_;
   std::unique_ptr<AliasDb> aliasDb_;
   std::shared_ptr<Graph> graph_;
+  using FusionCallback = std::function<bool(Node*)>;
+  FusionCallback callback_ = [&](Node* n) { return isFusableDefault(n); };
```

Comment:
Hmm can you really capture `this` in a lambda that's in a default initializer of a member? It's a surprising syntax.